conans/client/conf/__init__.py

```diff
@@ -22,12 +22,14 @@
         version: ["7.0", "7.1", "8.0", "8.1", "8.2", "8.3", "9.0", "9.1", "9.2", "9.3", "10.0", "10.1", "10.2", "10.3"]
     FreeBSD:
     SunOS:
-arch: [x86, x86_64, ppc64le, ppc64, armv6, armv7, armv7hf, armv8, sparc, sparcv9, mips, mips64]
+    Arduino:
+        board: ANY
```

Comment:
Arduino could be considered as an OS, there is a part of the final binary that is the Arduino "library", providing the high level abstractions for Arduino programs, similar to a very basic OS. I am not opposed to that.

The real problem might be that of the "target" setting, very difficult to model. I think the same, if there was an active community of Arduino/Conan users, or some official initiative from the Arduino devs, it would be easier.